website/docs/r/compute_reserved_capacity.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+
+subcategory: "Classic infrastructure"
+layout: "ibm"
+page_title: "IBM : compute_reserved_capacity"
+description: |-
+  Manages IBM Cloud compute reserved capacity
+---
+
+
+# ibm_compute_reserved_capacity
+Create, or updatea reserved capacity for your virtual server instance. With reserved capacity, you can control the physical host your virtual server instance is deployed to. For more information, about the reserved capacity, see [reserved groups](https://cloud.ibm.com/docs/virtual-servers?topic=virtual-servers-about-reserved-virtual-servers). 
```

Comment:
Edit these words 
1. updatea as update
2. virtual server instance as Virtual Server Instance (in 2 occurance)